utils/sidecar-evm.py

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env python
+from flask import Flask
+from web3 import Web3
```

Comment:
are we sure we want to load an entire library? I'd recommend to calculate the age of the last block in pure python